src/js/msp/MSPHelper.js

```diff
@@ -2481,25 +2526,49 @@ MspHelper.prototype.sendLedStripConfig = function(onCompleteCallback) {
             }
         }
 
-        for (let overlayLetterIndex = 0; overlayLetterIndex < led.functions.length; overlayLetterIndex++) {
-            const bitIndex = ledOverlayLetters.indexOf(led.functions[overlayLetterIndex]);
-            if (bitIndex >= 0) {
-                mask |= bit_set(mask, bitIndex + 12);
+        if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_46)) {
+
+            for (let overlayLetterIndex = 0; overlayLetterIndex < led.functions.length; overlayLetterIndex++) {
+                const bitIndex = ledOverlayLetters.indexOf(led.functions[overlayLetterIndex]);
+                if (bitIndex >= 0) {
+                    mask |= bit_set(mask, bitIndex + 12);
+                }
             }
-        }
 
-        mask |= (led.color << 18);
+            mask |= (led.color << 19);
 
-        for (let directionLetterIndex = 0; directionLetterIndex < led.directions.length; directionLetterIndex++) {
-            const bitIndex = ledDirectionLetters.indexOf(led.directions[directionLetterIndex]);
-            if (bitIndex >= 0) {
-                mask |= bit_set(mask, bitIndex + 22);
+            for (let directionLetterIndex = 0; directionLetterIndex < led.directions.length; directionLetterIndex++) {
+                const bitIndex = ledDirectionLetters.indexOf(led.directions[directionLetterIndex]);
+                if (bitIndex >= 0) {
+                    mask |= bit_set(mask, bitIndex + 23);
```

Comment:
23 instead of 22, a bug?

Reply:
No, these changes are intended. The overlay is expanded from 6bit to 7bit and I had to shift every offset after them. That's why I asked if I should allocate more like 8-9 bits for the overlay and function.
I will update the PR with an illustration and then you will understand how it works.

Reply:
@haslinghuis wait a bit, I might be stupid. I have to review my code. I might used wrong numbers.

Reply:
@haslinghuis see in the MSPHelper.js file line 1169.